### PR TITLE
Add mobile menu with DOM manipulation

### PR DESCRIPTION
**Pull request by mrjuda**
Mobile menu implemented with DOM manipulation (js)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,38 +20,35 @@
...
-            About
-          </a>
-          <a href="#Contact">
-            Contact
-          </a>
-        </nav>
+      <div class="name-logo">
+        <a id="name-logo" href="#">Jud&aacute; Teixeira</a>
       </div>
+      <nav class="menu-mobile-container hide">
```
Comment (HENRYKC24) [inline index.html:27]: - [x] Great job! You did almost everything in this project. I just want you to do more to get your mobile menu very identical to that of Figma's. 
image
## Yours
image
- Could you please **blur** the background so it looks like the Figma design? You can do that by adding something like `backdrop-filter: blur(8px);` to the overlay element.
- You can prevent the body from scrolling vertically when the menu pops up.
new
You may try adding this class to the body when the menu pops up:
```
  height: 100%;
  overflow: hidden;
  ```

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+// const nameLogo = document.querySelectorAll('.name-logo');
+// const contact = document.querySelectorAll('.contact');
+// function menuMobileShow(){
+//   menuMobileList.classList.add('.menu-mobile-show');
+// }
+// function menuMobileHide(){
+//   menuMobileList.classList.remove('.menu-mobile-show');
+// }
+// menuMobileHambBtn.addEventListener('click', menuMobileShow);
+// menuMobileHambBtn.addEventListener('click', menuMobileHide);
```
Comment (HENRYKC24) [inline scripts/main.js]: - [x] It seems you forgot to remove this block of commented code in your codebase. Make your code cleaner and less messy by removing commented code from your project.

**Review by HENRYKC24 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUESTED:recycle:
Good
## Hi @mrjuda :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mrjuda) [reply index.html:27]: Done, thank you!

File: scripts/main.js (hunk 2 above)
Comment (mrjuda) [reply scripts/main.js]: Done, thank you!

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @mrjuda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**